framework/include/executioners/Transient.h

```diff
@@ -205,6 +209,10 @@ class Transient: public Executioner
   //iteration count (which starts at 0), increment by 1.
   Real numPicardIts() { return _picard_it+1; }
 
+  // returned stepper *MUST* use n_startup_steps to return the result of
+  // computeInitialDT.
+  Stepper* buildIterationAdaptiveDT(double tol, int n_startup_steps);
```

Comment:
This is showing symptoms of a "leaky abstraction".

`Transient` should not know (or need to know) that `IterationAdaptiveDT` exists